apps/prairielearn/src/lib/question-render.sql

```diff
@@ -187,7 +187,6 @@ SELECT
   to_jsonb(ci) AS course_instance,
   to_jsonb(c) AS variant_course,
   to_jsonb(qc) AS question_course,
-  lgj.id AS grading_job_id,
```

Comment:
This was presumably removed because we have `to_jsonb(lgj) AS grading_job,` above, correct?

Is it possible to make the same change to `select_basic_submissions`? I didn't look to closely at where the data flows from there.

Reply:
Indeed. I checked the uses of `grading_job_id`, and the only reference to this column in reference to a submission object comes from different queries, mainly in the external grader handling side. The object returned by this query and the `select_basic_submissions` query is only used for rendering, and the only reference to `grading_job_id` in that context is the one replaced by `grading_job?.id`.